# Generated by Django 2.0.4 on 2018-12-18 15:54

import django.contrib.postgres.fields.jsonb
from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('feedback', '0002_auto_20180409_1505'),
    ]

    operations = [
        migrations.CreateModel(
            name='FrontendSecrets',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('public_secret', models.TextField(unique=True)),
                ('private_secret', models.TextField()),
                ('date_created', models.DateTimeField(auto_now_add=True)),
            ],
            options={
                'managed': True,
            },
        ),
        migrations.CreateModel(
            name='ReportedError',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('public_secret', models.TextField()),
                ('error', django.contrib.postgres.fields.jsonb.JSONField()),
                ('date_created', models.DateTimeField(auto_now_add=True)),
            ],
            options={
                'ordering': ['-date_created'],
                'managed': True,
            },
        ),
        migrations.CreateModel(
            name='ReportedEvent',
            fields=[
                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('public_secret', models.TextField()),
                ('EnvironmentError', django.contrib.postgres.fields.jsonb.JSONField()),
                ('date_created', models.DateTimeField(auto_now_add=True)),
            ],
            options={
                'ordering': ['-date_created'],
                'managed': True,
            },
        ),
    ]
